packages/service/tests/unit/conftest.py

```diff
@@ -94,3 +95,9 @@ def session_management_client(
 def single_session_reservation(mocker: MockerFixture) -> Mock:
     """Test fixture that creates a mock SingleSessionReservation."""
     return mocker.create_autospec(SingleSessionReservation)
+
+
+@pytest.fixture(scope="module")
```

Comment:
Let's stop adding scope=module to new fixtures. If something is worth caching between tests, it's usually worth caching between tests in multiple modules. 

These directory-name fixtures are lightweight, so it's not very important to cache them. The default of scope=function is fine unless there are wider-scoped fixtures that depend on them.

Reply:
Sure, removed the scope as module.